controllers/mongodb_tls_test.go

```diff
@@ -37,17 +39,58 @@ func TestStatefulSet_IsCorrectlyConfiguredWithTLS(t *testing.T) {
 	err = mgr.GetClient().Get(context.TODO(), types.NamespacedName{Name: mdb.Name, Namespace: mdb.Namespace}, &sts)
 	assert.NoError(t, err)
 
-	assertStatefulsetVolumesAndVolumeMounts(t, sts, mdb.TLSOperatorCASecretNamespacedName().Name, mdb.TLSOperatorSecretNamespacedName().Name, "")
+	assertStatefulSetVolumesAndVolumeMounts(t, sts, mdb.TLSOperatorCASecretNamespacedName().Name, mdb.TLSOperatorSecretNamespacedName().Name, "", "")
 }
 
-func assertStatefulsetVolumesAndVolumeMounts(t *testing.T, sts appsv1.StatefulSet, expectedTLSCASecretName string, expectedTLSOperatorSecretName string, expectedPromTLSSecretName string) {
+func TestStatefulSet_IsCorrectlyConfiguredWithTLSAndX509(t *testing.T) {
```

Comment:
Same as above comment - please camelCase.

Reply:
In this test file, all tests follow that pattern. Should I change all of them to something like: "TestStatefulSetIsCorrectlyConfiguredWithTLSAndX509"?

Reply:
yeah - let's do that while we are at it. Thanks